tests/test

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-sudo GOPATH=$GOPATH go test -v
+sudo GOPATH=${PWD}/../gopath go test -v $*
```

Comment:
I moved this change to a separate PR with more documentation: #791.
This was to be able to run a single test manually.
Also, when it is run through the topdir `./test`, `$GOPATH` is set by sourcing `./build`. But when a developer runs the test directly, `$GOPATH` might not be set to the correct path.